features/specs/minderjarige-ouder.feature

```diff
@@ -0,0 +1,244 @@
+# language: nl
+Functionaliteit: Minderjarige ouder
+  Gezag bepalen voor een minderjarige wanneer één of beide ouders minderjarig zijn.
```

Comment:
Na het lezen van de hele feature lijkt het over de moeder te gaan die minderjarig is. Is het een idee om te beginnen met het eenvoudige scenario dat moeder minderjarig is? En dan de erkenning introduceren? Dan is het wat gemakkelijker te volgen. Vooral het hele verhaal van Heidi dat de moeder een verklaring moet afleggen voor de erkenning kunnen we dan opbouwen, zodat de redenering duidelijk wordt.